tcms/rpc/tests/test_testexecution.py

```diff
@@ -37,6 +37,34 @@ def test_add_comment_with_pk_as_int(self):
         self.assertEqual(created_comment['comment'], first_comment.comment)
 
 
+class TestExecutionRemoveComment(APITestCase):
+    """Test TestExecution.remove_comment"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.execution = TestExecutionFactory()
+
+    def test_delete_all_comments_with_pk(self):
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "Hello World!")
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "More comments")
```

Comment:
You can use `tcms.core.helpers.comments.add_comment` instead of going through the API. See the implementation of the TestExecution.add_comment API method itself.